elasticdl/python/common/k8s_client.py

```diff
@@ -384,29 +376,22 @@ def create_ps(self, **kargs):
 
     def delete_master(self):
         logger.info("pod name is %s" % self.get_master_pod_name())
-        self.client.delete_namespaced_pod(
-            self.get_master_pod_name(),
-            self.namespace,
-            body=client.V1DeleteOptions(grace_period_seconds=0),
-        )
+        self.delete_pod(self.get_master_pod_name())
 
     def delete_worker(self, worker_id):
-        self.client.delete_namespaced_pod(
-            self.get_worker_pod_name(worker_id),
-            self.namespace,
-            body=client.V1DeleteOptions(grace_period_seconds=0),
-        )
+        self.delete_pod(self.get_worker_pod_name(worker_id))
 
     def delete_embedding_service(self, embedding_service_id):
```

Comment:
There is no embedding service pod now. All related methods can be deleted.

Reply:
OK